omod/src/main/java/org/openmrs/module/reporting/web/taglib/FormatTag.java

```diff
@@ -48,57 +49,57 @@
 public class FormatTag extends TagSupport {
 
 	public static final long serialVersionUID = 1L;
-	
+
 	private final Log log = LogFactory.getLog(getClass());
-	
+
 	private String var;
-	
+
 	private Object object;
-	
+
 	private Date date;
-	
+
 	private Integer conceptId;
-	
+
 	private Concept concept;
```

Comment:
There are lots of formatting changes in this pull request. Can you discard them?

Reply:
@dkayiwa I have updated the pull request to discard the formatting changes.